readme.md

```diff
@@ -2,4 +2,4 @@ README
 
 clean up setup documentation
 
-new performance tests
+update performance tests
```

Comment:
great update. will be much easier to add new tests

Reply:
awesome! thanks